test/rcases.lean

```diff
@@ -62,7 +62,7 @@ end
 
 example : true :=
 begin
-  obtain ⟨n, h, f⟩ : ∃ n : ℕ, n = n ∧ true,
+  obtain ⟨n : ℕ, h : n = n, f : true⟩ : ∃ n : ℕ, n = n ∧ true,
```

Comment:
Is this required now? It looks like a pretty clear regression.

Reply:
No, it's just there to test type ascriptions.